mcda/utils/utils_for_main.py

```diff
@@ -47,36 +47,41 @@ def check_config_error(condition: bool, error_message: str):
 
 def check_config_setting(condition_robustness_on_weights: bool,
                          condition_robustness_on_indicators: bool,
-                         mc_runs: int) -> (int, int):
+                         mc_runs: int, random_seed: int) -> (int, int):
```

Comment:
Can you try to wrap the method definition in two lines ,i.e., extending the lines upto 120 characters? 
I think that will improve the readability. As below we can see lines that are extended until 120.

```
def check_config_setting(condition_robustness_on_weights: bool, condition_robustness_on_indicators: bool,
                             mc_runs: int, random_seed: int) -> (int, int):
```